src/containers/HomeContainers/Home.js

```diff
@@ -0,0 +1,79 @@
+import React, { Component } from 'react'
+import { Text, FlatList, View, StyleSheet } from 'react-native'
+import { getStarredOfferingsData } from '../../api/offerings'
+import CourseCard from '../../components/Cards/CourseCard'
+
+class Home extends Component {
```

Comment:
Can you make this a functional component?

Reply:
fixed